### DISCUSSION TITLE
Need help regarding adding seed cluster in the same master cluster (having following error)

### DISCUSSION THREAD

**Discussion by aliabbas666** [kubermatic/kubermatic#10686] [Q&A]
Got this error when run following deployment file for seed cluster.
Error from server (seed validation request f081f201-c567-4b74-95b5-201fef6b8258 rejected: failed to get client for seed "kubermatic": failed to create restMapper: Get "https://127.0.0.1:6443/api?timeout=32s": dial tcp 127.0.0.1:6443: connect: connection refused): error when creating "seed.example.yaml": admission webhook "seeds.kubermatic.io" denied the request: seed validation request f081f201-c567-4b74-95b5-201fef6b8258 rejected: failed to get client for seed "kubermatic": failed to create restMapper: Get "https://127.0.0.1:6443/api?timeout=32s": dial tcp 127.0.0.1:6443: connect: connection refused

yaml file for seed cluster (kubeconfig secret is already created)

apiVersion: kubermatic.k8c.io/v1
kind: Seed
metadata:
  name: kubermatic
  namespace: kubermatic
spec:
  # these two fields are only informational
  country: FR
  location: Paris

  # List of datacenters where this seed cluster is allowed to create clusters in
  # In this example, user cluster will be deployed in eu-central-1 on AWS.
  datacenters: {}
#    hetzner-fsn1:
#      country: Finland
#      location: EU
#      spec:
#        hetzner:
#          datacenter: hel1-dc2
#          network: network-1
  # reference to the kubeconfig to use when connecting to this seed cluster
  kubeconfig:
    name: kubeconfig-cluster-example
    namespace: kubermatic

**Comment by xrstf** [comment]
You won't be able to use a kubeconfig that points to `127.0.0.1` (or localhost), as this name is resolved from within the Kubermatic pods on the master cluster (and so `localhost` will be the `kubermatic-webhook` pod).

Please choose a hostname tha unambiguously points to the seed cluster.
